README.md

```diff
@@ -33,6 +33,7 @@ Updating to a newer version of the SDK? See our [release notes](https://github.c
 ### Android
 
 - Android 5.0 (API level 21) and above
```

Comment:
Unrelated: once we release a version that uses Terminal Android SDK 3.X or later, this should be bumped to Android 8.0 (API level 26)